server/src/platform/mod.rs

```diff
@@ -1,8 +1,13 @@
 mod buffer;
-mod crypto;
-
+pub mod drbg;
 pub use buffer::*;
-pub use crypto::*;
+use drbg::{Drbg, Entropy};
+
+/// Provides access to cryptographic operations.
+pub trait Crypto: Drbg + Entropy {}
+
+// For all types that implement every Crypto sub trait, also implement the combined trait.
+impl<T> Crypto for T where T: Drbg + Entropy {}
```

Comment:
You can use blanket trait impls to package up your trait requirements so that you don't have to re-state them everywhere. It's not a problem yet, but if it starts getting to be a problem I'll suggest something like that. First, we should probably do one iteration of comments before we explore that, but I wanted to call it out here.

Reply:
I just removed it all together and replaced it with a structure containing individual fields.